Utility.cpp

```diff
@@ -1,15 +1,20 @@
 #include "Utility.h"
 
+#include "Character.h"
+#include "DragonSlayer.h"
+#include "Dragon.h"
+#include "Dwarf.h"
+#include "Paladin.h"
+#include "Item.h"
 #include "HelpfulItem.h"
 #include "DefensiveItem.h"
-#include "Character.h"
-
+#include "AttackItem.h"
```

Comment:
Some of these don't need to be included.
- [ ] REVISE

Reply:
still not fixed!

Reply:
ok done.